extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js

```diff
@@ -35,6 +36,13 @@ class AmpFlyingCarpet extends AMP.BaseElement {
     return false;
   }
 
+  onLayoutMeasure() {
+    const width = this.getWin()./*OK*/getComputedStyle(this.element).width;
+    this.vsync_.mutate(() => {
+      setStyle(this.container_, 'width', width);
+    });
```

Comment:
i think this should be used only for measurements right? - you should probably do this onLayoutCallback?